index.html

```diff
@@ -0,0 +1,858 @@
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--THREE-->
+          <div class="image-card">
+            <img
+              src="./images/merlin_144381195_afc81c81-6721-4fe7-beab-1c08a72f5c71-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--Third ROW -->
+          <!--ONE-->
+
+          <div class="image-card">
+            <img
+              src="./images/15kaliningrad1-threeByTwoSmallAt2X-v30.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+
+          <!--TWO-->
+          <div class="image-card">
+            <img
+              src="./images/00SCI-VENUS1-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--THREE-->
+          <div class="image-card">
+            <img
+              src="./images/merlin_144381195_afc81c81-6721-4fe7-beab-1c08a72f5c71-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="logo">
+        <div class="footer-top-line"></div>
+        <div>
+          <img src="./images/image2.png" height="40" alt="TNYT Logo" />
+        </div>
+      </div>
+      <div class="bottom-menu">
+        <div>
+          <ul>
+            <li>NEWS</li>
+            <li><a href="#">Home Page</a></li>
+            <li><a href="#">World</a></li>
+            <li><a href="#">Coronavirus</a></li>
```

Comment:
The footer does not look like the original one. It's totally different.
Clone Page             |  Original Page
:-------------------------:|:-------------------------:
![Screenshot from 2021-02-18 18-43-19](https://user-images.githubusercontent.com/44072711/108436598-aef56a00-7219-11eb-89cc-dd65aa1d461d.png) | ![Screenshot from 2021-02-18 18-43-31](https://user-images.githubusercontent.com/44072711/108436613-b7e63b80-7219-11eb-9492-a0dd90ab591c.png)

Reply:
It is the same I just am unable to style font  and line spacing to make look the same. please give suggestion of how to achive this.